### PR TITLE
Fix noVNC disconnected when clicking/dragging mouse too fast

### PR DESCRIPTION
**Pull request by trinhpham**
### Purpose of changes

I added apt preferences to use an updated version of x11vnc from Ubuntu 18.10. We'll need to remove that after upgrading Ubuntu version.

### Types of changes
_Put an `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### How has this been tested?

The noVNC connection should not be broken after some fast mouse clicks/drags

### CODE REVIEW

File: docker/x11vnc.pref [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Package: *
```
Comment (trinhpham) [inline docker/x11vnc.pref]: This file to make sure only x11vnc is from cosmic, others should be from bionic.
Ref: https://medium.com/@george.shuklin/how-to-install-packages-from-a-newer-distribution-without-installing-unwanted-6584fa93208f

File: release.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # Bash version should >= 4 to be able to run this script.
 
-IMAGE="budtmo/docker-android"
+IMAGE="${DOCKER_ORG:-budtmo}/docker-android"
```
Comment (trinhpham) [inline release.sh]: Should allow forks to automate their builds :)

File: supervisord.conf [hunk 3]
```diff
@@ -21,6 +21,7 @@ command=/usr/bin/x11vnc -display %(ENV_DISPLAY)s -nopw -forever -shared
 stdout_logfile=%(ENV_LOG_PATH)s/x11vnc.stdout.log
 stderr_logfile=%(ENV_LOG_PATH)s/x11vnc.stderr.log
 priority=2
+autorestart=true
```
Comment (trinhpham) [inline supervisord.conf]: Last chance to try if the crash still happen

File: release.sh (hunk 2 above)
Comment (budtmo) [reply release.sh]: If I understand correctly, the user can push on their own docker hub? should `:-budtmo` need to be deleted?

**Comment by budtmo** [comment]
Thank you for the PR @trinhpham !

File: release.sh (hunk 2 above)
Comment (trinhpham) [reply release.sh]: I keep that default value to not break your Travis-ci build.
Personally, I think we should respect the value `budtmo` :D

File: docker/Emulator_x86 [hunk 4]
```diff
@@ -7,6 +7,18 @@ LABEL maintainer "Budi Utomo <budtmo.os@gmail.com>"
 #=============
 WORKDIR /root
 
+#===========
+# Polyverse
```
Comment (trinhpham) [inline docker/Emulator_x86]: This commit can reduce the image size by 1GB.
You can take a look at [my docker repo](https://hub.docker.com/r/trinhpham/docker-android-x86-9.0/tags)
image

File: release.sh (hunk 2 above)
Comment (budtmo) [reply release.sh]: but it wont work, will it? because e.g. my DOCKER_ORG is budtmo and yours is trinhpham not trinhpham-budtmo. or do I miss something?

Comment (trinhpham) [reply release.sh]: It a trick of bash syntax to make a default variable (ref: https://unix.stackexchange.com/a/122848), so you'll have `budtmo` as default value.

Comment (trinhpham) [reply release.sh]: @budtmo do you need any more explanation?

Comment (budtmo) [reply release.sh]: Hi, sorry for late response, I will check your PR and merge it asap. Thank you again for the good work!

**Comment by budtmo** [comment]
Great work @trinhpham ! thank you again for the PR!

**Pull request merged**